docs/01-engage/02-accounts/03-contact-details/01-enable-contact-preferences/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Enable Contact Preferences
+sidebar_position: 1
+---
+
+Contact is simply a way that involves meeting or communicating with anyone. So, when a contact wants to receive communication by any means, whether it is by telephone or email we call it a **contact preference**. This is important in terms of knowing and maintaining a valuable relationship with the donor or sponsor (the one you are contacting). 
+
+## Contact Preferences in ENGAGE
+
+ENGAGE allows you to prioritize your contact preferences while creating an account, be it individual or by an organization. There are in total four different types of preferences that ENGAGE provides:
+
+- Telephone
+- Email
+- SMS
+- Post
+
+Each preference contains options that clarify if a contact wants to be contacted for a specific purpose or not.
```

Comment:
Then something like each channel contains one or more categories of communication. Common categories are "Marketing", "Administration", "Events" etc. but the exact list will be configured by your system administration.

For each specific way and purpose of communication, Engage is able to record whether the account has opted in to such communication, opted out, or if the preference is unspecified (e.g. unknown).